packet_packer.go

```diff
@@ -67,6 +67,11 @@ type coalescedPacket struct {
 	shortHdrPacket *shortHeaderPacket
 }
 
+// IsShortHeaderPacket says if this packet only contains a short header packet (and no long header packets).
+func (p *coalescedPacket) IsShortHeaderPacket() bool {
+	return len(p.longHdrPackets) == 0 && p.shortHdrPacket != nil
+}
```

Comment:
NIT: Better name `IsOnlyShortHeaderPacket`

Reply:
I like it!